src/dev_deps.ts

```diff
@@ -7,3 +7,7 @@ export {
 } from "https://deno.land/std@0.132.0/testing/asserts.ts";
 export * as mf from "https://deno.land/x/mock_fetch@0.3.0/mod.ts";
 export { isHttpError } from "https://deno.land/std@0.182.0/http/http_errors.ts";
+export {
+  afterEach,
+  beforeAll,
+} from "https://deno.land/std@0.185.0/testing/bdd.ts";
```

Comment:
nit: These are script test dependencies, how come they're included in `dev_deps` but the script dependencies aren't?

Reply:
Bth the question of where to put the script dependencies was more involved then I thought

- The script dependencies could not be included in `scripts/deps.ts` because the scripts folder is not actually a `module` all the script operate independently, and may require different Deno permissions
- They could not be included in `dev_deps` since the scripts are not a module with `dev_deps` importers, other imports in `dev_deps` could potentially required additional deno permissions (ex: `--allow_env`) which the scripts do not require
- The dependencies for the scripts `tests` could be included in `dev_deps` since they are executed alongside the other `tests` in the module

I think we can leave it like this for now, we can change this in later PR if this causes issues, the changes are not part of the Deno package